docs/_news/news10.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news
+title: New for Codewind 0.10.0
+description: New for Codewind 0.10.0
+keywords: news, new, updates, update, version, hybrid
+duration: 1 minute
+permalink: news10
+---
+
+## Codewind 0.10.0
+Friday, 20 March 2020
+
+#### ✨ New Features and Highlights for 0.10.0 ✨
+**IntelliJ Tech Preview Updates**
+
+We've continued to work hard on getting the IntelliJ experience up and consistent with our other IDEs. Here are the latest IntelliJ updates:
+
+- [View project log files in IntelliJ](https://github.com/eclipse/codewind/issues/989).
+- Open and [view the performance dashboard](https://github.com/eclipse/codewind/issues/2038) for select project types.
+- [Enable and disable projects](https://github.com/eclipse/codewind/issues/987).
+- [Simultaneously select and remove multiple projects](https://github.com/eclipse/codewind/issues/2225).
+- Create a new project with the [**New Project** menu action](https://github.com/eclipse/codewind-intellij/pull/64).
+![](images/imagesfornews/newprojintellij.gif){:width="800"}
+
+**IDEs (Both Eclipse and VS Code)**
+- The [Performance Dashboard is always enabled](https://github.com/eclipse/codewind/issues/2299) even if the project does not have metrics available. 
+    - Note that for [projects without metrics, running loads fail](https://github.com/eclipse/codewind/issues/2384).
+
+**VS Code**
+- You can use the profiling extension for [remote connections](https://github.com/eclipse/codewind/pull/2123) and in [Che](https://github.com/eclipse/codewind-vscode/pull/457).
+- [The **add to workspace** project command is reworked for multi-root workspaces](https://github.com/eclipse/codewind-vscode/pull/484). It no longer appears on projects that are already in the root workspace folder and gives an overrideable warning if run on a project that is in a subfolder of the workspace.
+- The homepage is updated so that [the **create** and **add** project creation steps are greyed out and unclickable until the actions are allowed](https://github.com/eclipse/codewind/issues/2255). On the **Local** tab, local codewind must be started. On the **Remote** tab, [a remote connection must exist](https://github.com/eclipse/codewind/issues/2279).
+- The **Remote Connection** settings page layout is [improved and more responsive](https://github.com/eclipse/codewind-vscode/pull/476).
+- [Miscellaneous corrections have been made to Webview links and buttons](https://github.com/eclipse/codewind-vscode/pull/473)
```

Comment:
-  "were made" instead of "have been made"
- Should webview be lower case?
- Period at end of sentence.

Reply:
not sure if webview should be capitalized or not. @jopit ?

Reply:
The VSCode documentation does not capitalize webview https://code.visualstudio.com/api/extension-guides/webview so we should follow their lead.